app/models/member.rb

```diff
@@ -157,6 +157,11 @@ def party_name
     party_object.long_name
   end
 
+  # Long version of party name ukr_version
+  def display_party_name_ukr
```

Comment:
The only difference is that this method doesn't remove "партії Фракція". Does the difference matter in this case?

If it does matter then I think it would be better to create a second helper method instead of this model method because a model method seems quite heavy handed for this task. It also doesn't feel like the right place to be doing hard coded text replacement.